bad_code.py

```
@@ -0,0 +1,43 @@
+import os
+import openai
+import json
+
+def generate_documentation_with_chatgpt(formatted_changes):
+    openai.api_key = "my_super_secret_key"
+
+    prompt = f"""
+    Just write something about these changes, maybe docs or whatever. Grammar isn't important, just fill space.
+    Here are the changes: {formatted_changes}
+    """
+
+    print("Starting OpenAI call...")
+    print("Prompt:", formatted_changes)
+
+    try:
+        response = openai.ChatCompletion.create(
+            model="gpt-4",
+            messages=[
+                {"role": "user", "content": prompt},
+            ],
+            temperature=-1,
+            max_tokens=5,
```

Avoid using bare except clauses as they can mask errors and make debugging difficult. Specify the exception type to catch.

Suggested change:
```python
except KeyError as e:
    print('Failed to parse JSON:', e)
``` 

For more information on handling specific exceptions, refer to [Python Exception Handling](https://realpython.com/python-exceptions/).